src/CLIMA-atmos/Dycore/drivers/rising_thermal_bubble.jl

```diff
@@ -27,134 +22,159 @@ macro hascuda(ex)
   return HAVE_CUDA ? :($(esc(ex))) : :(nothing)
 end
 
-meshgenerator(part, numparts, Ne, dim, DFloat) =
-brickmesh(ntuple(j->range(DFloat(0); length=Ne+1, stop=1000), dim),
-          (true, ntuple(j->false, dim-1)...),
-          part=part, numparts=numparts)
+using ParametersType
+using PlanetParameters: R_d, cp_d, grav, cv_d
+@parameter gamma_d cp_d/cv_d "Heat capcity ratio of dry air"
+@parameter gdm1 R_d/cv_d "(equivalent to gamma_d-1)"
```

Comment:
> @parameter gamma_d cp_d/cv_d "Heat capcity ratio of dry air"
@parameter gdm1 R_d/cv_d "(equivalent to gamma_d-1)"

Again, these parameters can be moved to PlanetParamers (or into a hyper-parameter list if that is more appropriate).

Reply:
I'll skip commenting on the remaining ones, but there are several terms like this across many files it seems.

Reply:
They could be added to `PlanetParameters` if you wanted. (Maybe in a separate pull request?)

Since they are derived parameters, I could go either way. We mainly packed them into `@parameter` types in order to get compile time type conversions.

Reply:
Thanks for looking through the code.